src/index.html

```diff
@@ -1,17 +1,184 @@
+            >
+          </label>
+        </div>
+      
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input 
+              type="date" 
+              name="dateOfBirth" 
+              value="mm/dd/yyyy"
+            >
+          </label>
+        </div>
+      
+        <div class="terms">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" name="terms">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration:</legend>
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input 
+              type="email" 
+              name="email" 
+              placeholder="email@example.com" 
+              required
+            >
+          </label>
+        </div>
+      
+        <div class="password">
+          <label>
+            Password:
+            <input 
+              type="password" 
+              name="password" 
+              required
+            >
+          </label>
+        </div>
+    </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cat">
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="cat">
+            No
+          </label>
+        </div>
+      
+        <div class="form-field">
+          <label for="color">What is your favorite color?
+            <input 
+              type="color" 
+              name="color" 
+              value="#000000"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input 
+              type="time" 
+              name="timeToBed" 
+              step="2"
+            >
+          </label>
+        </div>
+      
+        <div class="form-field">
+          <label for="carBrand">What are your favorite brands of cars?
+            <select name="carBrand" multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="lada">lada</option>
+            </select>
+          </label>
+        </div>
+      
+        <div class="work">
+          <label>
+            How do you rate our work?
+            <input 
+              type="range" 
+              name="rate" 
+              min="0" 
+              max="100"
+              value="0"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label for="comments">Comments:
+            <textarea class="comment" 
+          name="comments" 
+          minlength="50" 
+          maxlength="300"
+          >
+            </textarea>
```

Comment:
Follow the code style from the checklist, please 
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/99403198/234837196-2849a299-0d21-4678-ab21-7d9aa71d7265.png">